data/mapping-data.json

```diff
@@ -425,14 +356,22 @@
                     "mapping_set_group": "mondo_mappings",
                     "registry_confidence": null,
                     "local_name": null,
-                    "mapping_set_version": "http://purl.obolibrary.org/obo/mondo/releases/2025-02-04/mappings/mondo_hasdbxref_hp.sssom.tsv",
+                    "mapping_set_version": "http://purl.obolibrary.org/obo/mondo/releases/2025-04-01/mappings/mondo_hasdbxref_hp.sssom.tsv",
                     "mapping_set_title": "No title available",
                     "mapping_set_description": "No description available",
                     "mapping_provider": "No provider information available",
                     "issue_tracker": "No issue tracker available",
                     "license": "https://creativecommons.org/publicdomain/zero/1.0/"
                 }
             ]
+        },
+        {
+            "registry_id": "mesh-mappings",
```

Comment:
@rays22 you have not actually updated your mapping registry file yet! https://raw.githubusercontent.com/mapping-commons/mesh-mappings/main/mappings.yml - that needs to look a lot more like https://github.com/mapping-commons/mh_mapping_initiative/blob/master/registry.yml